sdk/src/client/core.rs

```diff
@@ -104,20 +106,16 @@ impl ClientInner {
         // create invalid transactions/blocks.
         #[cfg(target_family = "wasm")]
         {
-            lazy_static::lazy_static! {
```

Comment:
For the team: do we have other instances of similar things that wouldn't play nicely with more than one instance?

Reply:
Doesn't look like it, but we do have this weird lazy_static macro that seems to have been a custom replacement for the lib:

```rust
#[cfg(feature = "mqtt")]
macro_rules! lazy_static {
    ($init:expr => $type:ty) => {{
        static mut VALUE: Option<$type> = None;
        static INIT: std::sync::Once = std::sync::Once::new();

        INIT.call_once(|| unsafe { VALUE = Some($init) });
        unsafe { VALUE.as_ref() }.expect("failed to get lazy static value")
    }};
}
```